cogs/commands/karma.py

```diff
@@ -473,13 +485,32 @@ async def plot(self, ctx: Context, *, args: str):
         file = File(img, filename=filename)
         await ctx.send(f"Here you go, {name}! {emoji}", embed=embed, file=file)
 
+    @karma.command(help="Plots the karma change over time of the given karma topic(s)")
+    @commands.cooldown(5, 60, BucketType.user)
+    async def plot(self, ctx: Context, *, args: str):
+        await self.common_plot(ctx, args, False)
+
     @plot.error
     async def plot_error_handler(self, ctx: Context, error: KarmaError):
         if hasattr(error, "message"):
             await ctx.send(error.message)
         else:
             raise error
```

Comment:
This is not directly related, but when testing removing this raise made the error handling fallback to global correctly. Would be great if we could just roll that in
```suggestion
```